README.md

```diff
@@ -1 +1,61 @@
-# SENG 499 - Company 2 - Algorithm 1 
\ No newline at end of file
+# SENG 499 - Company 2 - Algorithm 1 
+
+This repository contains a package, [Course-Scheduler](https://pypi.org/project/coursescheduler/), for
+that returns a course schedule for three terms of Software Engineering courses at UVic. The Course Scheduler uses the 
+Constraint Satisfaction Problem (CSP) Algorithm to determine the hard constraint of the schedule, and produces a schedule 
+that adheres to the constraints. The scheduler also takes into account soft constraints for professor preferences, 
+however, these constraints are not always fulfilled. 
```

Comment:
I'd suggest keeping this brief and not getting into hard and soft constraints since it can be a bit confusing without context. I do think we should include some documentation on the design of our solution in the repo, but it could probably go in its own markdown file and it's out of scope for this ticket.

```suggestion
This repository contains a package ([_coursescheduler_](https://pypi.org/project/coursescheduler/)) that schedules the courses required for a software engineering degree at University of Victoria.
```

If you think we should include some high-level design details, I'd suggest this:
```suggestion
This repository contains a package ([_coursescheduler_](https://pypi.org/project/coursescheduler/)) that schedules the courses required for a software engineering degree at University of Victoria. The problem is implemented as a constraint satisfaction problem and an optimized backtracking search algorithm is used to find a valid schedule.
```